ruby/vampires.rb

```diff
@@ -1,8 +1,4 @@
-
-likes_garlic_bread = true
-wants_insurance = true
 current_year = 2017
```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.